sdks/python/tox.ini

```diff
@@ -27,44 +28,78 @@ select = E3
 
 # Shared environment options.
 [testenv]
-# Set [] options for pip install, e.g., pip install apache-beam[test].
+# Set [] options for pip installation of apache-beam tarball.
 extras = test
-# Always recreate the virtual environment.
-recreate = True
-# Pass these environment variables to the test environment.
-passenv = TRAVIS*
 # Don't warn that these commands aren't installed.
 whitelist_externals =
   find
   time
 
 [testenv:py27]
 commands =
+  python --version
+  pip --version
   {toxinidir}/run_tox_cleanup.sh
   python apache_beam/examples/complete/autocomplete_test.py
-  python setup.py test
+  #python setup.py test
   {toxinidir}/run_tox_cleanup.sh
 
 [testenv:py27-cython]
```

Comment:
What is py27-cython, py27-cython2, py27-cython3 ?

Reply:
Please ignore. Testing.